corehq/apps/export/templates/export/customize_export_new.html

```diff
@@ -167,6 +167,26 @@
                             value: sharing
                         ">
                     </select>
+                    {% if has_other_owner %}
+                    <div data-bind="visible: sharing() != initialSharing()" class="alert alert-warning">
```

Comment:
Can you replace `.alert.alert-warning` with `.help-block` and use `.has-warning` on the `.form-group` container when this warning is visible? Alert classes aren't meant to be used as form field feedback - the alert border right under to the select box without any spacing looks weird.

Reply:
updated